packages/frontend-2/components/dashboard/ProjectCard.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div
+    class="bg-foundation border border-1 border-outline-3 rounded-xl p-5 pb-4 flex flex-col gap-y-3"
+  >
+    <NuxtLink
+      :to="projectRoute(project.id)"
+      class="text-heading hover:text-primary truncate"
+    >
+      {{ project.name }}
+    </NuxtLink>
+    <div class="flex-1">
+      <p class="text-body-3xs text-foreground-2 capitalize">
+        {{ project.role?.split(':').reverse()[0] }}
+        <span class="pl-1 pr-2">•</span>
+        <span v-tippy="updatedAt.full">
+          {{ updatedAt.relative }}
+        </span>
+      </p>
+    </div>
+    <UserAvatarGroup :users="teamUsers" :max-count="4" />
+    <div>
+      <FormButton
+        :to="allProjectModelsRoute(project.id) + '/'"
```

Comment:
Why the trailing slash? The route helper should already format the route as needed. If it doesn't, then the helper needs changing, not the return of it